model.js

```diff
@@ -73,4 +99,17 @@ const updateVotes = (review_id, inc_votes) => {
       return {review : review[0].rows}
    })
 }
-module.exports = {fetchCategories, fetchReviews, fetchReviewById, fetchComments, newComment, updateVotes, fetchAllUsers}
+
+const checkCategories = (category) => {
+   if(!category){
+      return 'No Category'
+   }
+   const categoryStr = category.replace('_', ' ')
+   return db.query(`SELECT * FROM reviews WHERE category = '${categoryStr}'`).then((categories)=>{
+      if (categories.rows.length === 0){
+         return Promise.reject({status:400, msg:'Invalid Category'})
+      }
+      else return category
```

Comment:
Great that you're checking the DB for categories! This way it'll work if the data changes or any other categories are added ⭐